python/cuml/cuml/kernel_ridge/kernel_ridge.pyx

```diff
@@ -247,6 +255,7 @@ class KernelRidge(Base, RegressorMixin):
                                 filter_params=True, **params)
 
     @generate_docstring()
+    @enable_device_interop
     def fit(self, X, y, sample_weight=None,
```

Comment:
Also, small detail, but it is always nice to use the `self.n_features_in_` attribute instead of local variable `n_rows` when ingesting data. This gives one more thing to replicate sklearn's behavior.